addons/mail/static/src/models/composer/composer.js

```diff
@@ -597,11 +597,12 @@ function factory(dependencies) {
         /**
          * @private
          * @param {string} content html content
-         * @returns {string|undefined} command, if any in the content
+         * @returns {mail.channel_command|undefined} command, if any in the content
          */
         _getCommandFromText(content) {
             if (content.startsWith('/')) {
-                return content.substring(1).split(/\s/)[0];
+                const user_input = content.substring(1).split(/\s/)[0];
```

Comment:
Mmm you already had your mind in week-end I think. This is still not camel case, I guess I'll do it so we can merge.